 <launch>
<!-- joy node -->
  <node respawn="true" pkg="joy"
        type="joy_node" name="motor_joy" >
    <param name="dev" type="string" value="/dev/input/js0" />
    <param name="deadzone" value="0.12" />
  </node>

 <!-- Axes -->
<!-- change it to be accoring to the Lienz platform --> 
  <param name="axis_linear_y" value="0" type="int"/>
  <param name="axis_linear_x" value="1" type="int"/>
<param name="axis_roational_z" value =" 3" type="int"/>
 
<!--param name="button_angular_plus" value="4" type="int"/ -->
<!--param name="button_angular_minus" value="5" type="int"/-->
  <param name="scale_linear_x" value="0.2" type="double"/>
  <param name="scale_linear_y" value="0.2" type="double"/>
  <param name="scale_angular" value="0.4" type="double"/>
  <node pkg="epos_ethercat" type="Joy_publisher" name="teleop"/>
</launch>

